core/overlays/stage/configmaps.yaml

```diff
@@ -58,3 +58,10 @@ data:
   instance-workflow-controller-metrics-backend: "workflow-controller-metrics-thoth-backend-stage.apps.ocp.prod.psi.redhat.com:80"
   instance-workflow-controller-metrics-middletier: "workflow-controller-metrics-thoth-middletier-stage.apps.ocp.prod.psi.redhat.com:80"
   instance-workflow-controller-metrics-amun-inspection: "workflow-controller-metrics-thoth-amun-inspection-stage.apps.ocp.prod.psi.redhat.com:80"
+---
```

Comment:
please include a minimun configmap in here to https://github.com/thoth-station/thoth-application/blob/master/core/base/configmaps.yaml
```
kind: ConfigMap
apiVersion: v1
metadata:
  name: qeb-hwt-workflow
```
as the stage and test configmap, get patched to the base, and if we dont have the configmap in th base we would have issue in argocd applying kustomize

Reply:
Added, thanks!